app/db/AtomWorkshopDB.scala

```diff
@@ -0,0 +1,48 @@
+package db
+
+import com.amazonaws.services.dynamodbv2.model.AmazonDynamoDBException
+import com.gu.atom.data.{DynamoCompositeKey, DynamoDataStore}
+import com.gu.contentatom.thrift.{Atom, AtomData, AtomType, ContentChangeDetails}
+import com.gu.contentatom.thrift.atom.cta.CTAAtom
+import com.gu.contentatom.thrift.atom.explainer.{DisplayType, ExplainerAtom}
+import com.gu.contentatom.thrift.atom.media.MediaAtom
+import play.api.Logger
+import cats.syntax.either._
+
+object AtomWorkshopDB {
+
+  def buildKey(atomType: AtomType, id: String) = DynamoCompositeKey(atomType.name, Some(id))
+
+  def buildDefaultAtom(atomType: AtomType) = {
+    val defaultAtoms: Map[AtomType, AtomData] = Map(
+      AtomType.Explainer -> AtomData.Explainer(ExplainerAtom("-", "-", DisplayType.Flat)),
+      AtomType.Cta -> AtomData.Cta(CTAAtom(""))
```

Comment:
No because eventually we won't support media atoms I think - I included them in the other bit because currently the only test atom we have is a media one.